src/frontends/tensorflow_common/src/op/boolvariant.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <memory>
+
+namespace ov {
+
+class BoolVariant {
+public:
+    BoolVariant(bool value) : value_(value) {}
+
+    bool get() const {
+        return value_;
+    }
+
+    static std::shared_ptr<BoolVariant> make(bool value) {
+        return std::make_shared<BoolVariant>(value);
+    }
+
+private:
+    bool value_;
+};
+
+}  // namespace ov
```

Comment:
we do not need any auxiliary classes to support complex type.
Please see example PR: https://github.com/openvinotoolkit/openvino/pull/21477
You just need to update existing `translate_round_op` and propagate `ComplexTypeMark`.

Reply:
Hii @rkazants can you tell me how to test my code?